### ISSUE TITLE
[AXE-CORE]: Heading levels should only increase by one

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#4210]
### [AXE-CORE]: Heading levels should only increase by one

The `h2` **Stories** is followed by a series of `h4` starting with **Battlefield Acupuncture comes to VA Pittsburgh**. 

The same invalid heading relationship is in the next area with the `h2` **Events** followed by the `h4` **Veterans Town Hall**.

## Type of Issue
Template

#### Disabilities Affected: 
* Blind
* Deafblind
* Mobility

#### Why It Matters
The underlying purpose of headers is to convey the structure of the page. For sighted users, the same purpose is achieved using different sizes of text. Text size, however, is not helpful for users of screen readers, because a screen reader identifies a header only if it is properly marked-up. When heading elements are applied correctly, the page becomes much easier to navigate for screen reader users and sighted users alike.

In the same way that sighted users can glance at a page and get a sense of its contents, users of screen readers can do the same by navigating through headings. Well written and properly ordered headings can save users, especially those who use screen readers, a lot of time and frustration.

The purpose of headings is to describe the structure of the webpage, not just highlight important text. They should be brief, clear, unique, and marked with h1 through h6 elements applied in hierarchical order. All of these qualities make headings valuable tools for screen reader users. Similar to the way sighted users can glance at a page and get a sense of its contents, screen reader users can navigate through headings. Well written and properly ordered headings can save screen reader time and frustration.

In addition to making the page more accessible, headings have other benefits since search engines use headings when filtering, ordering, and displaying results. Improving the accessibility of your site can also have the effect of making your page more findable.

## Point of Contact

Jennifer

## Acceptance Criteria

As a screen reader user, I want to navigate the content hierarchy context on the page so that I may understand the material of interest.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: all
* Server destinations: 
  * https://staging.va.gov/pittsburgh-health-care/
  * https://www.va.gov/pittsburgh-health-care/events/
  * https://www.va.gov/pittsburgh-health-care/events/past-events/
  * https://www.va.gov/pittsburgh-health-care/stories/
  * https://www.va.gov/pittsburgh-health-care/news-releases/
  * https://www.va.gov/pittsburgh-health-care/performance/
  * https://www.va.gov/pittsburgh-health-care/psychology-internships-and-fellowships/
  * https://www.va.gov/pittsburgh-health-care/events/mindful-mondays-2/ (and other event detail views)

## Steps to Recreate

1. Enter `https://staging.va.gov/pittsburgh-health-care/` in browser
2. Inspect the code in the *Stories* section
3. Verify the `h2` is currently followed by two `h4` instead of `h3`s
4. To further understand the impact, use a screen reader to navigate the content order

### Current Code
```html
<section>
            <h2 class="vads-u-margin-bottom--3 medium-screen:vads-u-margin-top--5" id="stories">Stories</h2>
            
    <div class="region-list usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
    <div class="region-grid vads-u-margin-right--2 ">
        <h4 class=" vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--0p5"><a href="/pittsburgh-health-care/stories/battlefield-acupuncture-comes-to-va-pittsburgh">Battlefield Acupuncture comes to VA Pittsburgh</a></h4>
        <p class="vads-u-margin-y--0">Veteran Nadine Stanford was in excruciating pain. Twenty minutes later, after a series of 10 small needles were placed in her ear, the pain was gone. Battlefield acupuncture is how it happens.</p>
    </div>
    <div class="region-grid  vads-u-order--first small-screen:vads-u-order--initial vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0">
        <img class="region-img" src="https://staging-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/3_2_medium_thumbnail/public/2019-11/battlefield-accupuncture.jpg" alt="Battlefield Accupuncture" title="" width="480" height="320">
    </div>
</div>

<div class="region-list usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
    <div class="region-grid vads-u-margin-right--2 ">
        <h4 class=" vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--0p5"><a href="/pittsburgh-health-care/stories/new-sim-center-improves-veterans-health-care">New Sim Center Improves Veterans Health Care</a></h4>
        <p class="vads-u-margin-y--0">VA Pittsburgh Healthcare System's new state-of-the-art simulation center serves a critical role in improving Veterans health care.</p>
    </div>
    <div class="region-grid  vads-u-order--first small-screen:vads-u-order--initial vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0">
        <img class="region-img" src="https://staging-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/3_2_medium_thumbnail/public/2019-09/SIM-Main-2.jpg" alt="Grand opening guests view an emergence delirium simulation." title="" width="480" height="320">
    </div>
</div>

                        <a class="usa-button usa-button-secondary vads-facility-hub-button vads-u-width--full medium-screen:vads-u-width--auto vads-u-font-size--md vads-u-margin--0" href="/pittsburgh-health-care/stories" id="ab0f55f00ac50daef75f6ca0bd952651">
              See all stories <i class="fa fa-chevron-right va-l-font-size--12px"></i>
            </a>
          </section>
```

### Recommended Fix
```html
<section>
            <h2 class="vads-u-margin-bottom--3 medium-screen:vads-u-margin-top--5" id="stories">Stories</h2>
            
    <div class="region-list usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
    <div class="region-grid vads-u-margin-right--2 ">
        <h3 class=" vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--0p5"><a href="/pittsburgh-health-care/stories/battlefield-acupuncture-comes-to-va-pittsburgh">Battlefield Acupuncture comes to VA Pittsburgh</a></h3>
        <p class="vads-u-margin-y--0">Veteran Nadine Stanford was in excruciating pain. Twenty minutes later, after a series of 10 small needles were placed in her ear, the pain was gone. Battlefield acupuncture is how it happens.</p>
    </div>
    <div class="region-grid  vads-u-order--first small-screen:vads-u-order--initial vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0">
        <img class="region-img" src="https://staging-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/3_2_medium_thumbnail/public/2019-11/battlefield-accupuncture.jpg" alt="Battlefield Accupuncture" title="" width="480" height="320">
    </div>
</div>

<div class="region-list usa-grid usa-grid-full vads-u-margin-bottom--3 medium-screen:vads-u-margin-bottom--4 vads-u-display--flex vads-u-flex-direction--column small-screen:vads-u-flex-direction--row">
    <div class="region-grid vads-u-margin-right--2 ">
        <h3 class=" vads-u-margin-top--0 medium-screen:vads-u-margin-bottom--0p5"><a href="/pittsburgh-health-care/stories/new-sim-center-improves-veterans-health-care">New Sim Center Improves Veterans Health Care</a></h3>
        <p class="vads-u-margin-y--0">VA Pittsburgh Healthcare System's new state-of-the-art simulation center serves a critical role in improving Veterans health care.</p>
    </div>
    <div class="region-grid  vads-u-order--first small-screen:vads-u-order--initial vads-u-margin-bottom--2 medium-screen:vads-u-margin-bottom--0">
        <img class="region-img" src="https://staging-va-gov-assets.s3-us-gov-west-1.amazonaws.com/img/styles/3_2_medium_thumbnail/public/2019-09/SIM-Main-2.jpg" alt="Grand opening guests view an emergence delirium simulation." title="" width="480" height="320">
    </div>
</div>

                        <a class="usa-button usa-button-secondary vads-facility-hub-button vads-u-width--full medium-screen:vads-u-width--auto vads-u-font-size--md vads-u-margin--0" href="/pittsburgh-health-care/stories" id="ab0f55f00ac50daef75f6ca0bd952651">
              See all stories <i class="fa fa-chevron-right va-l-font-size--12px"></i>
            </a>
          </section>
```

## WCAG or Vendor Guidance (optional)

* [axe-core 3.4 - Heading levels should only increase by one](https://dequeuniversity.com/rules/axe/3.4/heading-order?application=AxeChrome)

## Screenshots

![Screen Shot 2020-02-04 at 3.55.02 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/39152b85-4f0f-4421-909a-8fb3918e79de)

---
![Screen Shot 2020-02-04 at 4.21.37 PM.png](https://images.zenhubusercontent.com/5ac217b74b5806bc2bcd3fc8/d1ff651e-3b08-45ab-a964-929b43e6c594)

**Comment by mmiddaugh** [comment]
Per @jenstrickland 's comments on #3978 

> For reference, I opened this issue #6097, and assigned it to the VSP sitewide team, during the audit, as an axe-core issue was found in the page template, that heading levels should only increase by one. This is due to there only being an <h1> on this page, and the footer content starts with an <h4>. This is outside the scope of the VSA Facilities team's efforts.

I will tag this issue similarly

**Comment by jenstrickland** [comment]
@mmiddaugh This is not the same issue. #6097 is related to footer content. This ticket refers to content within the main area of the page.

**Comment by davidconlon** [comment]
We need to update these h3 tags instead to h2 tags and use the font size utility to get the proper (same) sizing as h3

Ref:
https://design.va.gov/design/typography#headings-and-accessibility
https://design.va.gov/utilities/font-size

**Comment by mmiddaugh** [comment]
@jenstrickland this one is ready for your review

**Comment by jenstrickland** [comment]
Validated. 🎉
